invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/RunButton.js

```diff
@@ -0,0 +1,116 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 CERN
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { http } from "react-invenio-forms";
+import {
+  Button,
+  Dropdown,
+  DropdownMenu,
+  Form,
+  FormField,
+  FormInput,
+  TextArea,
+} from "semantic-ui-react";
+
+export class RunButton extends Component {
+  constructor(props) {
+    super();
+
+    this.state = {
+      title: "Manual run",
+      config: JSON.stringify(props.config, null, "\t"),
+      queue: "low",
+      loading: false,
+    };
+  }
+
+  handleChange = (e, { name, value }) => this.setState({ [name]: value });
```

Comment:
minor suggestion: you can be explicit about what is being changed to avoid unexpected side-effects:

```javascript
handleConfigChange(...)
handleQueueChange(...)
handleTitleChange(...)
```

I know it's duplicating code for now but it's easier to read and to extend (e.g. imagine you want to add some validation when you change `config` without having to modify the code for the others states)